### ISSUE TITLE
Grunt uglify

### ISSUE THREAD

**Issue by osv** [angular/material#5210]
This or maybe other commit within last 1day broke my grunt ulgify task:
https://github.com/angular/material/commit/275e604cdcc576567b5a33c25307d3b6bb35bc2d

I got missing semicolon here
```
window.ngMaterial={version:{full:"0.11.4-master-275e604"}};(function(a)
----------------------------------------------------------^
```

**Comment by osv** [comment]
Maybe you should just add semicolon at end :)
```js
.pipe(insert.append('window.ngMaterial={version:{full: "' + VERSION +'"}};'))
```

**Comment by jelbourn** [comment]
Fixed via 17f4bf49b268b473619b2253b009e721b13cd797
